### ISSUE TITLE
Problem with option babel.transpileModules and scoped packages

### ISSUE THREAD

**Issue by Gaboa** [egoist/poi#547]
``` json
"poi": {
  "babel": {
      "transpileModules": [
        "@<scope>/<packageName>"
      ]
    }
}
```
With this config it will not transpile the code.
But with packages without scope it works well.
Poi version 12.5.8

**Comment by egoist** [comment]
Are you using Windows?

**Comment by egoist** [comment]
Should be fixed in 12.6.3
